tests/test_aaa_authorization_service.rb

```diff
@@ -92,6 +95,14 @@ def tacacs_groups
     %w(tac_group bxb100 sjc200 rtp10)
   end
 
+  def valid_cfg?(cfg)
+    skip('tacacs_server.yaml file is empty') unless cfg
+    msg = 'Missing key in tacacs_server.yaml'
```

Comment:
Also, please specify the full relative path to the `tacacs_server.yaml` file so users of the tests know exactly where it is to quickly and easily populate it.